pkg/agent/clean/node_notwindows.go

```diff
@@ -0,0 +1,590 @@
+// +build !windows
+
+package clean
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/rancher/rke/hosts"
+	"github.com/sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	DockerPipe = "/var/run/docker.sock"
+	HostMount  = "/host"
+)
+
+func Run(ctx context.Context, args []string) error {
+	if len(args) > 3 {
+		fmt.Println(usage())
+		return nil
+	}
```

Comment:
Any reason why we aren't returning an error?

Reply:
also, if `len(args) < 3`, why is `Node(ctx)` (cleaning?) the default behavior? Seems like a weird thing to do if the args are `agent job`.

Reply:
cause it prints the usage and assumes you dont know what youre doing? possible usage should be the default behavior... let me thing about this.